### PR TITLE
Remove sekizai dependency

### PR DESCRIPTION
**Pull request by yakky**
By using some tricks we can make sekizai not required anymore in a backward compatible way

### CODE REVIEW

**Comment by yakky** [comment]
@m-vdb @tezro could you have a look at this? By using ``<html {% meta_namespaces_gplus %}>`` you should be able to use google+ metatags without the need for sekizai

File: cms_helper.py [hunk 1]
```diff
@@ -15,11 +14,18 @@
...
 
+try:
+    import sekizai  # NOQA
+
+    HELPER_SETTINGS['INSTALLED_APPS'].append('sekizai')
+    HELPER_SETTINGS['TEMPLATE_CONTEXT_PROCESSORS'] = [
+        'sekizai.context_processors.sekizai',
+    ]
+except ImportError:
+    pass
```
Comment (m-vdb) [inline cms_helper.py]: love it ❤️

File: meta/templatetags/meta.py [hunk 2]
```diff
@@ -2,11 +2,43 @@
...
 from django import template
+from django.conf import settings
 from django.utils.html import escape
+from django.utils.safestring import mark_safe
 from django.utils.six import string_types
 
 register = template.Library()
 
+try:
+    from django.apps import apps
```
Comment (m-vdb) [inline meta/templatetags/meta.py]: why can't it be imported at toplevel?

Comment (yakky) [inline meta/templatetags/meta.py]: django.apps only exists in Django 1.7+, this package currently supports also Django 1.6

Comment (yakky) [inline meta/templatetags/meta.py]: eh eh eh :)
I'll drop 1.6 after the new release

Comment (m-vdb) [inline meta/templatetags/meta.py]: cool :) and thank you so much for the quick workaround!

**Pull request merged**